### DISCUSSION TITLE
Data tracking site

### DISCUSSION THREAD

**Discussion by pglevy** [Bixal/rapid-response-team#21] [Ideas]
Something for standing up a data aggregation site quickly, like the [COVID Tracking Project](https://covidtracking.com/). Some are trying to do this [now in India](https://covid19.neera.ai/?tab=delhiAvailableBeds&profile=covid_backup).

![image](https://user-images.githubusercontent.com/5177337/116104010-1c8c9f00-a67e-11eb-84c0-3cc990843bce.png)

**Comment by ghost** [comment]
- a single site location 
- spin up a central location that aggregates data from different places to quickly make info available in a more accessible way

**Comment by bnaolu** [comment]
Really like this one. Post-Covid, it can be a great tool to used for hospitals, etc. for data tracking (thinking long-term, but maybe not within our realm?)
